airflow/cli/cli_parser.py

```diff
@@ -778,6 +783,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.dag_command.dag_list_dags'),
         args=(ARG_SUBDIR, ARG_OUTPUT),
     ),
+    ActionCommand(
+        name='generate_yaml',
```

Comment:
What do you think to add this to the new ``kubernetes`` group? We currently have sections for celery executor.

```
Celery components

  airflow celery flower - Start a Celery Flower
  airflow celery stop - Stop the Celery worker gracefully
  airflow celery worker - Start a Celery worker node
```

Reply:
@mik-laj so create a section for one command? Seems like a bit of overkill...

Reply:
Ok. In the future, we can move this to a separate group.